src/dcqc/file.py

```diff
@@ -334,7 +334,9 @@ def stage(
             if self._local_path is not None:
                 return self._local_path
             else:
-                destination_str = mkdtemp()
+                # TODO: This prefix is used by nf-dcqc to easily find the staged file.
```

Comment:
+1 on this TODO:.  Will file a jira ticket for this:

                # TODO: This prefix is used by nf-dcqc to easily find the staged file.
                #       It might be worth using a DCQCTMPDIR to avoid hard-coding this.
                destination_str = mkdtemp(prefix="dcqc-staged-")